odata-parser.pegjs

```diff
@@ -711,14 +701,9 @@ QuotedTextBind =
 
 boundary =
 	(	&'('
-	/	space+
+	/	[\ +]+
 	)
 
 spaces =
-	space*
+	[\ +]*
 
-space =
```

Comment:
Fwiw I don't mind leaving this as an alias to avoid issues where in future people may add an extra thing that wants to check for spaces but don't realize that in a url a `+` can be equivalent to a space and so only check `' '` instead of `[ +]`. Unless of course there's a significant performance benefit